src/routes/fetchTree.tsx

```diff
@@ -31,11 +31,7 @@ async function handleDataFetch(targetUrl: string, dispatch: Function) {
   try {
     response = await axios.get(targetUrl);
   } catch {
-    const errorMessage =
-      `Attempted to fetch data from URL: ${targetUrl}, but ` +
-      "there was a problem with fetching the data. Please confirm " +
-      "the URL is publicly accessible and has the appropriate CORS policy " +
-      "in place. You may want to look in your browser console.";
+    const errorMessage = `We weren't able to import your tree data. Please confirm this URL is correct and publicly accessible: ${targetUrl}`;
```

Comment:
This is more human-readable, but I do kind of want the bread crumbs about CORS... I foresee that being an issue that might get encountered and can be really flummoxing if you have never seen a problem like that before. Although I guess their browser console would at least mention CORS. I suppose the real thing I want is for this to flag to the user, "hey, something went wrong in a way you probably have the power to fix, but will require a reasonable amount of technical know-how." Being able to take the `targetUrl` value, paste it in to a browser, and get JSON is _not enough_ to guarantee Galago can get it, and that's by the design of how browsers work.

I have no idea how to unpack all of that meaning into a short error message, so maybe this approach is the way to go, but that's what I was thinking.

Reply:
Yeah I generally agree. My thinking here was largely that the intended audience for Galago is largely folks who aren't technical... like at all. I barely (read: mostly don't) even know what CORS is, so I'm kind of assuming that users won't either / I'm not sure pointing them in that direction would actually help. 

I'm going to split the difference -- leave the error as-is for now, and I'll include an FAQ entry about "I see JSON at the end of my URL why doesn't fetch work" and link it from the error once it exists.

Reply:
Yeah, it's also quite possible that the best course of action is for someone to think our site is broken and reach out to us. I don't know if we'll see that many users clamoring for help, and we would have the technical background to resolve it (hopefully) even if it's not technically _our_ bug.